CONTRIBUTOR.md

```diff
@@ -0,0 +1,152 @@
+# Contributing to Iroh
+
+We'd love for you to contribute to our source code and to make Iroh even better.
+
+Here are the guidelines we'd like you to follow:
+
+* [Code of Conduct](#coc)
+* [Questions and Problems](#question)
+* [Issues and Bugs](#issue)
+* [Feature Requests](#feature)
+* [Improving Documentation](#docs)
+* [Issue Submission Guidelines](#submit)
+* [Pull Request Submission Guidelines](#submit-pr)
+
+## <a name="coc"></a> Code of Conduct
```

Comment:
Doesn't github already automatically make headings clickable? Or is this to give them a consistent name?